### ISSUE TITLE
Error during Antenna build

### ISSUE THREAD

**Issue by FeatureIDE** [FeatureIDE/FeatureIDE#431]
```
java.lang.NoSuchFieldError: ruleMemo
	at antenna.preprocessor.v3.parser.APPParser.<init>(APPParser.java:89)
	at antenna.preprocessor.v3.parser.Defines.addDefines(Defines.java:197)
	at antenna.preprocessor.v3.Preprocessor.addDefines(Preprocessor.java:325)
	at de.ovgu.featureide.antenna.AntennaPreprocessor.performFullBuild(AntennaPreprocessor.java:144)
	at de.ovgu.featureide.core.builder.ExtensibleFeatureProjectBuilder.build(ExtensibleFeatureProjectBuilder.java:172)
	at org.eclipse.core.internal.events.BuildManager$2.run(BuildManager.java:734)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:42)
	at org.eclipse.core.internal.events.BuildManager.basicBuild(BuildManager.java:205)
	at org.eclipse.core.internal.events.BuildManager.basicBuild(BuildManager.java:245)
	at org.eclipse.core.internal.events.BuildManager$1.run(BuildManager.java:300)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:42)
	at org.eclipse.core.internal.events.BuildManager.basicBuild(BuildManager.java:303)
	at org.eclipse.core.internal.events.BuildManager.basicBuildLoop(BuildManager.java:359)
	at org.eclipse.core.internal.events.BuildManager.build(BuildManager.java:382)
	at org.eclipse.core.internal.events.AutoBuildJob.doBuild(AutoBuildJob.java:144)
	at org.eclipse.core.internal.events.AutoBuildJob.run(AutoBuildJob.java:235)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
```

**Comment by meinicke** [comment]
The old Antenna jar contained an old versions of antlr classes that were used for preprocessing. These classes were overloaded by another antlr version in fm.core which is incompatible with the Antenna implementation (call to a non existing field).
I created a new repository for Antenna (see:[Antenna](https://github.com/FeatureIDE/Antenna)) that uses the same antlr version as in fm.core.
I also removed a lot of unnecessary code so the jar is much smaller now.
